packages/react-core/src/components/ToggleGroup/ToggleGroup.tsx

```diff
@@ -9,6 +10,8 @@ export interface ToggleGroupProps extends React.HTMLProps<HTMLDivElement> {
   className?: string;
   /** Modifies the toggle group to include compact styling. */
   isCompact?: boolean;
+  /** Disable all toggle group items under this component. */
+  disableAll?: boolean;
```

Comment:
the naming doe not quite match our usual conventions.  maybe `areAllDisabled` or `areAllGroupsDisabled`

Reply:
Yep, this is much better!